recipes/decore/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "decore" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2aff5856b50e1650dd7b2294722994a359ea05966c2303dca9b017d97219f9aa
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - decore
```

can we try to add ``pip check`` here please?
```suggestion
    - decore
  requires:
    - pip
  commands:
    - pip check
```